menu/src/components/MainPage/MainPageList.tsx

```diff
@@ -228,6 +228,11 @@ export const MainPageList: React.FC = () => {
   };
 
   const handleDeleteVehicle = () => {
+    // If onesync is disabled, show an error due to server side entity handling
+    if (!serverCtx.oneSync.status) {
+      return enqueueSnackbar(t('nui_menu.page_main.delete_veh.onesync_error'), { variant: 'error' })
```

Comment:
Lets just have a combined locale key for both this and delete vehicle as they are practically the same.

Reply:
Pushed a commit combining them into one locale.